proposals/lip-0042.md

```diff
@@ -134,11 +173,19 @@ The default reward of the block as a 64-bit unsigned integer.
 
 #### After Block Execution
 
-After a block `b` is executed, the `mint` function exposed by the [token module][research:token-module] to assign the reward to the block generator as:
+After a block `b` is executed, the `mint` function exposed by the [Token module][token-module] to assign the reward to the block generator as:
 
 ```python
-blockReward = getBlockReward(b.header)
+(blockReward, reduction) = getBlockReward(b.header)
 mint(b.header.generatorAddress, TOKEN_ID_REWARD, blockReward)
```

Comment:
Should we maybe only call mint here if the reward is positive?

Reply:
Good point, I added a check.